src/main/java/io/jenkins/docker/DockerTransientNode.java

```diff
@@ -35,6 +36,8 @@
 
     private String cloudId;
 
+    private AtomicBoolean acceptingTasks = new AtomicBoolean(true);
```

Comment:
I think a simple `boolean` (perhaps `volatile`) would suffice here.

Reply:
a `volatile boolean` would then be equivalent